Benchmarks/Schema.NET.Benchmarks/SchemaBenchmarkBase.cs

```diff
@@ -0,0 +1,31 @@
+namespace Schema.NET.Benchmarks
+{
+    using System;
+    using BenchmarkDotNet.Attributes;
+    using BenchmarkDotNet.Jobs;
+    using Newtonsoft.Json;
+
+    [SimpleJob(RuntimeMoniker.NetCoreApp30)]
```

Comment:
There are a few more useful atrributes that could be added here to generate some graphs and more stats. See https://github.com/Dotnet-Boxed/Framework/blob/master/Benchmarks/Boxed.Mapping.Benchmark/MapListBenchmark.cs

Reply:
Yep, happy to add those. Do you think it is worth benchmarking multiple versions of .NET like your linked example or just leave it still as .NET Core 3?

Reply:
.NET Framework is still used a fair bit, so I added it. The results can be very different.